pkg/apis/computenode/v1alpha1/computenodeopenstack_types.go

```diff
@@ -24,6 +24,17 @@ type ComputeNodeOpenStackSpec struct {
 	InfraDaemonSets []InfraDaemonSet `json:"infraDaemonSets,omitempty"`
 	// Nodes to delete upon scale down
 	NodesToDelete []NodeToDelete `json:"nodesToDelete,omitempty"`
+        // Image used for drain pod which performs compute removal, this is usually
```

Comment:
this does not match with the CR example, this (L27-37) should not be part of nodes to delete but global, right?